packages/core/core/src/services/document-service/repository.ts

```diff
@@ -318,11 +318,13 @@ export const createContentTypeRepository: RepositoryFactoryMethod = (uid) => {
     return { versions: draftEntries };
   }
 
-  async function updateComponents(entry: any, data: any) {
-    return components.updateComponents(uid, entry, data);
+  async function updateComponents(opts = {} as any) {
```

Comment:
I don't understand the change here 🤔 the point is that you shouldn't need to pass an uid to the updateComponents at all because it's already in context and that it should just receive and entry and some data I don't think you should be changing that code.

Reply:
Good point, thanks!

Reply:
Sorry I still don't get the change ? why transform this into an object 🤔

Reply:
You're right sorry I think I was misunderstanding how something was working with `packages/core/core/src/services/document-service/middlewares/middleware-manager.ts`

Does this look better?